src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -888,6 +888,9 @@ internal bool TryGetContainingEditableSpan(SnapshotPoint point, out SnapshotSpan
             return false;
         }
 
+        internal bool IsInOpenTextBuffer(SnapshotPoint point)
+            => _openTextBuffers.ContainsKey(point.Snapshot.TextBuffer);
```

Comment:
This might be my lack of context, but when I read `IsInOpenTextBuffer(SnapshotPoint)` I think "Is this point in any open text buffer". Realistically this method is actually "Is this buffer open", so I think it would make more sense to have the parameter here be a `ITextBuffer` and put the `.Shapshot.TextBuffer` bit in the caller on line 79.

Reply:
Sort of. Technically it's "Is this point in any open buffer that is being tracked". The point is still in an open buffer, but it's tracked by a separate workspace when doing a preview.